src/main/java/com/fasterxml/jackson/core/util/Separators.java

```diff
@@ -97,26 +114,30 @@ public Separators(
             Spacing objectFieldValueSpacing,
             char objectEntrySeparator,
             Spacing objectEntrySpacing,
+            String objectEmptySeparator,
             char arrayValueSeparator,
-            Spacing arrayValueSpacing
+            Spacing arrayValueSpacing,
+            String arrayEmptySeparator
```

Comment:
This changes signature, no longer same method added in 2.16; I'll change it to chain to new method